### ISSUE TITLE
Unable to configure two trackers (that have been working for months)

### ISSUE THREAD

**Issue by xeviff** [Jackett/Jackett#12169]
### Do not delete the template.

**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue.
Check closed issues as well, because your issue may have already been fixed. Also check our [Troubleshooting](https://github.com/Jackett/Jackett/wiki/Troubleshooting) for steps to resolve common issues.

Please read our [Contributing Guidelines](https://github.com/Jackett/Jackett/blob/master/CONTRIBUTING.md) before submitting your issue to ensure a prompt response to your bug.

### Environment
Synology
**OS**:
Docker
**.Net Runtime**: [.Net-Core/.Net-Framework/Mono]
idk
**.Net Version**:
idk
**Jackett Version**:
v0.18.545
**Last Working Jackett Version**:
same
**Are you using a proxy or VPN?** [yes/no]
no
### Description
[List steps to reproduce the error and details on what happens and what you expected to happen]
Suddently I'm not able to connect sonarr/raddar to a couple of indexes (trackeros and red-bits). If I access to the Torznab URL, it returns "Invalid API Key", But if click the "Test" button on the indexer it passes the verification succesfully

I double checked the login parameters and all is correct. I don't know where to start for solve it...

### Logged Error Messages
[Place any relevant error messages you noticed from the logs here.]
[Make sure you attach the full logs with your personal information removed in case we need more information]
nothing referred

### Screenshots
[Place any screenshots of the issue here if needed]
![image](https://user-images.githubusercontent.com/73612508/129482967-1576779c-edae-49da-b425-566f5e05e1b9.png)
![image](https://user-images.githubusercontent.com/73612508/129483094-7d239014-b68e-4aae-9218-d73a58df5035.png)

**Comment by garfield69** [comment]
turn on the enhanced logging on the jackett dashboard and click the apply server settings button, then access the sonarr indexer and perform a test.
If you then examine the Jackett log you will see the torznab request that sonarr sent to jackett and the response back.
you can also then check the sonarr logs to see what sonarr though of the response it got back from jackett.
this should go some way to helping you begin to figure out what is happening.

**Comment by xeviff** [comment]
Ok, I found this error stacktrace in Radarr log :

[v3.2.2.5080] System.Net.WebException: The SSL connection could not be established, see inner exception.: 'https://jackett.xxx.yyy/api/v2.0/indexers/trackeros/results/torznab/api?t=caps&apikey=(removed) ---> System.Net.WebException: The SSL connection could not be established, see inner exception.
 ---> System.Net.Http.HttpRequestException: The SSL connection could not be established, see inner exception.
 ---> System.Security.Authentication.AuthenticationException: The remote certificate was rejected by the provided RemoteCertificateValidationCallback.
   at System.Net.Security.SslStream.SendAuthResetSignal(ProtocolToken message, ExceptionDispatchInfo exception)
   at System.Net.Security.SslStream.ForceAuthenticationAsync[TIOAdapter](TIOAdapter adapter, Boolean receiveFirst, Byte[] reAuthenticationData, Boolean isApm)
   at System.Net.Security.SslStream.ProcessAuthentication(Boolean isAsync, Boolean isApm, CancellationToken cancellationToken)
   at System.Net.Security.SslStream.AuthenticateAsClient(SslClientAuthenticationOptions sslClientAuthenticationOptions)
   at System.Net.Http.ConnectHelper.EstablishSslConnectionAsyncCore(Boolean async, Stream stream, SslClientAuthenticationOptions sslOptions, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---

I changed my reserve proxy in order to work through http instead of https, and it worked.
Since is not a Jackett problem I close the ticket. Thanks for your time!
